packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js

```diff
@@ -148,12 +148,12 @@ const FiatConvertor = (props) => {
       return (
         <LimitsHelper>
           {
-            form === 'coinifyCheckoutBuy'
-              ? <FormattedMessage id='buy.quote_input.remaining_buy_limit' defaultMessage='Your remaining buy limit is {max}' values={{ max: <a onClick={() => setMax(limits.max)}>{curr}{limits.max}</a> }} />
-              : getSellLimits()
+            isSell
+              ? getSellLimits()
+              : <FormattedMessage id='buy.quote_input.remaining_buy_limit' defaultMessage='Your remaining buy limit is {max}' values={{ max: <a onClick={() => setMax(limits.max)}>{curr}{limits.max}</a> }} />
           }
           {
-            level.name < 2 && !equals(prop('state', kyc), 'reviewing')
+            gt(2, prop('name', level)) && not(equals(prop('state', kyc), 'reviewing'))
```

Comment:
Looks like the line taken out reads as "level less than 2 AND kyc does not equal reviewing"
The new line reads "level greater than 2 AND kyc does not equal reviewing"

Is the change intended here, and why? The only users who should see the increase limits message are level 1.

Reply:
No, the new line does reads 2 greater than name of level and kyc not reviewing. See https://ramdajs.com/docs/#gt

Reply:
Ah, that's my bad. Will approve this then.